LICENSE.txt

```diff
@@ -1,31 +1,31 @@
-Unless noted below, Minecraft Forge, Forge Mod Loader, and all 
+Unless noted below, Minecraft NeoForge, NeoForge Mod Loader, and all 
 parts herein are licensed under the terms of the LGPL 2.1 found
 here http://www.gnu.org/licenses/old-licenses/lgpl-2.1.txt and 
 copied below.
 
-Homepage: http://minecraftforge.net/
-          https://github.com/MinecraftForge/MinecraftForge
+Homepage: https://neoforged.net/
+          https://github.com/neoforged/NeoForge
           
 
 A note on authorship:
 All source artifacts are property of their original author, with
 the exclusion of the contents of the patches directory and others
 copied from it from time to time. Authorship of the contents of
-the patches directory is retained by the Minecraft Forge project.
+the patches directory is retained by the Minecraft NeoForge project.
```

Comment:
The project name is NeoForge, not Minecraft NeoForge

Reply:
Should be fixed with newest commit, thanks!